docs/docker-integration/tutorial-swarm-compose.rst

```diff
@@ -92,22 +92,6 @@ The Docker Compose files both have the same layout, as illustrated below, except
 
   * Flocker will automatically provision this volume on-demand if it doesn't already exist.
 
-Step 2.1: Optionally specify size and profile
----------------------------------------------
```

Comment:
I removed this whole section in favour of adding an example of the plugin options into the sample configuration file. 
I think the options are self-explanatory.
These old commands no-longer work because `docker-compose` adds a prefix to the names of the volumes it expects (or creates).
These old commands would only work if we specified `external` volumes in the compose file.
 * https://docs.docker.com/compose/compose-file/#volume-configuration-reference